### PR TITLE
chore: remove popconfirm

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Towards https://github.com/PostHog/posthog/issues/13624

## Changes

Replace all instances of `Popconfirm` with a `LemonDialog` (or an appropriate tooltip in one case)

## How did you test this code?

Checked every instance. Some examples

|Before|After|
|----|----|
| Screenshot 2024-03-08 at 15 24 18 | Screenshot 2024-03-08 at 15 34 27 |

Tooltip with no delay
Screenshot 2024-03-08 at 15 11 25

Feature flag toggle example
https://github.com/PostHog/posthog/assets/6685876/3d434adc-f3d8-4405-9bc3-f2b941280e58

### CODE REVIEW

File: frontend/src/scenes/appContextLogic.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import * as Sentry from '@sentry/react'
```
Comment (benjackwhite) [inline frontend/src/scenes/appContextLogic.ts:1]: I think this is a merge error - should be removed.

File: frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ export function SessionRecordingsFilters({
                     className="w-full rounded-none border-0 border-t"
                     multiple
                     defaultActiveKeys={initiallyOpen ? ['advanced-filters'] : []}
-                    size="small"
+                    size="xsmall"
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx:97]: ?

**Review by benjackwhite — APPROVED** [review]
Other than the accidental commit it looks good to me

File: frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx (hunk 2 above)
Comment (daibhin) [reply frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx:97]: Weird hangover from something else I was experimenting with. Removing in next commit

**Pull request merged**